bin/helm-docs-diff

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
+set -eu
+
+dir_dirty=$(git diff HEAD)
```

Comment:
What protoc-diff does is that it invokes `protoc-go.sh` directly and then checks to see if the state is dirty.  Can we do a similar thing here were we invoke `bin/helm-docs` in this script rather than in a separate workflow step.

Reply:
Absolutely, fixed.